packages/portalnetwork/src/subprotocols/history/history.ts

```diff
@@ -152,7 +160,40 @@ export class HistoryProtocol extends BaseProtocol {
                     getHistoryNetworkContentId(1, 4),
                     decoded.value as Uint8Array
                   )
+                  break
                 }
+                // case HistoryNetworkContentTypes.HeaderProof: {
```

Comment:
Do we need any of this any more or can it be removed?

Reply:
Nope -- deleted